source/_integrations/nest.markdown

```
@@ -163,24 +163,78 @@ Now that you have authentication configured, you will create a Nest Device Acces
 Read the warnings on the page before proceeding, including Google Account type limitations.
 {% endnote %}
 
-1. Click on the button **[Go to the Device Access Console](https://console.nest.google.com/device-access/)**.
+2. Click on the button **[Go to the Device Access Console](https://console.nest.google.com/device-access/)**.
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)